GiniVision/Classes/Core/Logger.swift

```diff
@@ -7,40 +7,28 @@
 
 import Foundation
 
-final class Logger {
-    
-    enum Event {
-        case error
-        case success
-        case warning
-        case custom(emoji: String)
-        
-        var value: String {
-            switch self {
-            case .error: return "❌"
-            case .success: return "✅"
-            case .warning: return "⚠️"
-            case .custom(let emoji): return emoji
-            }
-        }
-    }
+enum LogEvent: String {
+    case error = "❌"
+    case success = "✅"
+    case warning = "⚠️"
+}
+
+func Log(message: String,
+         event: LogEvent,
+         giniConfig: GiniConfiguration = .shared) {
+    Log(message: message, event: event.rawValue, giniConfig: giniConfig)
+}
+
+func Log(message: String,
+         event: String,
+         giniConfig: GiniConfiguration = .shared) {
```

Comment:
No one would pass a configuration object to a logger method, I think. Can you make it a property in the class?

Reply:
I don't understand, what's that class?

Reply:
I wanted to say the Logger class. But it's no longer there.
I'm not sure why you removed it though. You could have kept it and had a `Log` method that uses it.